src/geode/geosciences/gocad_common.cpp

```diff
@@ -399,6 +399,108 @@ namespace geode
             file << "END_ORIGINAL_COORDINATE_SYSTEM" << EOL;
         }
 
+        PropHeaderData read_prop_header( std::ifstream& file )
+        {
+            PropHeaderData header;
+            auto opt_line =
+                geode::detail::goto_keyword_if_it_exists( file, "PROPERTIES" );
+            if( !opt_line )
+            {
+                geode::Logger::info(
+                    "Token PROPERTIES could not be found in the file, "
+                    "attributes will not be loaded." );
+                return header;
+            }
+            auto line = opt_line.value();
+            std::vector< absl::string_view > split_line = absl::StrSplit(
```

Comment:
```suggestion
            auto split_line = absl::StrSplit(
```
is it enough for your case?

Reply:
No, I need to convert it implicitly to a vector to get its size